examples/counter/common/build.gradle

```diff
@@ -0,0 +1,97 @@
+repositories {
+    maven { url "https://dl.bintray.com/spekframework/spek-dev" }
+}
+apply plugin: 'java'
+apply plugin: 'kotlin-multiplatform'
+
+
+kotlin {
+    jvm()
+    js() {
+        [compileKotlinJs, compileTestKotlinJs].each { configuration ->
+            configuration.kotlinOptions {
+                moduleKind = 'umd'
+                sourceMap = true
+                metaInfo = true
+            }
+        }
+    }
+
+
+    iosArm64("ios")
+    iosX64("iosSim")
+    macosX64("macos")
+    mingwX64("win")
+    wasm32("wasm")
+    linuxArm32Hfp("linArm32")
+    linuxMips32("linMips32")
+    linuxMipsel32("linMipsel32")
+    linuxX64("lin64")
+
+    sourceSets {
+        commonMain {
+            dependencies {
+                implementation kotlin("stdlib-common")
+                implementation project(":lib")
+            }
+        }
+        commonTest {
+            kotlin.srcDir('src/test/kotlin')
+            dependencies {
+                implementation kotlin("test-common")
+                implementation kotlin("test-annotations-common")
+                implementation "org.spekframework.spek2:spek-dsl-metadata:$project.versions.spek"
+                implementation "ch.tutteli.atrium:atrium-cc-en_GB-robstoll-common:$project.versions.atrium"
+                implementation "io.mockk:mockk-common:1.9.3"
+            }
+        }
+
+        jvmMain {
+            kotlin.srcDir('src/jvmMain/kotlin')
+            dependencies {
+                implementation kotlin("stdlib")
+            }
+        }
+        jvmTest {
+            dependencies {
+                implementation kotlin("test")
+                implementation kotlin("test-junit")
+                implementation "org.spekframework.spek2:spek-dsl-jvm:$project.versions.spek"
+                implementation "ch.tutteli.atrium:atrium-cc-en_GB-robstoll:$project.versions.atrium"
+                implementation "io.mockk:mockk:1.9.3"
+
+                runtimeOnly "org.spekframework.spek2:spek-runner-junit5:$project.versions.spek"
+                runtimeOnly 'org.jetbrains.kotlin:kotlin-reflect'
+            }
+        }
+        jsMain {
+            kotlin.srcDir('src/jsMain/kotlin')
+            dependencies {
+                implementation kotlin("stdlib-js")
+            }
+            compileKotlinJs {
+                kotlinOptions.metaInfo = true
+                kotlinOptions.sourceMap = true
+                kotlinOptions.suppressWarnings = true
```

Comment:
Are you actually getting warnings now that you'd like to ignore? Otherwise it's probably best to be strict and keep the project clean by treating warnings as errors

Reply:
Good idea.  have not got into the JS side much yet.  Will remove.  Yes, I agree with treating warnings as errors.